addons/website_sale/models/website.py

```diff
@@ -78,6 +78,9 @@ def _get_product_sort_mapping():
         ],
         default='optional')
 
+    hide_add_to_cart = fields.Boolean(string="Hide 'Add To Cart' when price = 0")
+    contact_us_button_url = fields.Char(string="Contact Us Button URL")
```

Comment:
Maybe we can add a default to `/contactus` ?

Reply:
Done